src/solc_0.7/catalyst/AssetMinter.sol

```diff
@@ -123,7 +123,7 @@ contract AssetMinter is WithMetaTransaction {
                 require(catalystsQuantities[assets[i].catalystId] != 0, "INVALID_CATALYST_NOT_ENOUGH");
                 uint16 maxGems = _gemsCatalystsRegistry.getMaxGems(assets[i].catalystId);
                 require(assets[i].gemIds.length <= maxGems, "INVALID_GEMS_TOO_MANY");
-                catalystsQuantities[assets[i].catalystId]--;
+                catalystsQuantities[assets[i].catalystId]--; // 1 should 1 UNIT, see CATALYST_UNIT ?
```

Comment:
@wighawag I don't think we need the UNITS here as we're just operating on function args, not erc20 contract state.

Reply:
That's right, this contract do not deal with token balances at all

Reply:
The issue is that we use that values as is in ` _gemsCatalystsRegistry.batchBurnGems(from, ids, gemsQuantities);` below

Reply:
I mean in `_batchBurnCatalysts` for catalysts
and `_bachBurnGems` for gems

Reply:
Ah yes. I see it. Will push a fix for review tomorrow.